front_end/src/utils/errors.ts

```diff
@@ -16,8 +18,17 @@ export function extractError(field_error: any): string | undefined {
 }
 
 export function logError(error: Error | unknown, message?: string) {
-  Sentry.captureException(error);
-  console.error(message ?? error);
+  const statusCode = (error as ErrorResponse)?.response?.status;
+  const digest = (error as ErrorResponse)?.digest;
+
+  if (!statusCode || statusCode >= 500) {
+    Sentry.captureException(error);
+  }
+
+  console.error(
```

Comment:
Want to make it similar to the heroku logs which is very easy to parse & perform lookup:
<img width="984" alt="image" src="https://github.com/user-attachments/assets/e1003fc9-6f06-449b-adf8-8cb21138986d" />